tsconfig.json

```diff
@@ -36,6 +36,7 @@
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
     /* Module Resolution Options */
+    "resolveJsonModule": true,
```

Comment:
Nice. Does this also include that json file into the bundle that is created when building apk/ipa?